apps/api_web/lib/api_web/controllers/shape_controller.ex

```diff
@@ -100,22 +100,19 @@ defmodule ApiWeb.ShapeController do
       ShapeResource:
         resource do
           description("""
-          Shape representing the stops to which a particular trip can go. Trips grouped under the same route can have \
-          different shapes, and thus different stops. See \
-          [GTFS `shapes.txt`](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#shapestxt)
+          Sequence of geographic points representing a path vehicles will travel on a trip. See \
+          [GTFS `shapes.txt`](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#shapestxt).
           """)
 
           attributes do
             polyline(:string, """
-            ## Encoding/Decoding
+            The sequence of points in \
+            [Encoded Polyline Algorithm Format](https://developers.google.com/maps/documentation/utilities/polylinealgorithm).
+            Libraries for decoding polylines are available in many languages, for example:
 
-            [Encoded Polyline Algorithm Format](https://developers.google.com/maps/documentation/utilities/polylinealgorithm)
-
-            ## Example Libraries
-
-            * [Javascript](https://www.npmjs.com/package/polyline)
-            * [Erlang](https://blog.kempkens.io/posts/encoding-and-decoding-polylines-with-erlang/)
```

Comment:
Is this Erlang post out of date?

Reply:
I didn't think it was very relevant since it's sort of a how-to for writing your own polyline decoder, rather than a library you can install (which I think is the point here: that libraries exist and you probably _shouldn't_ build your own).